_posts/2020-11-07-What-is-Tmux.md

```diff
@@ -0,0 +1,87 @@
+
+**Tmux interface: on the command line, plain text**
+
+
+## Think About It This Way
+Let's say a student and their team must complete a presentation in two days, but this student has to leave and cannot access the slides after the first day. In order to complete the task on time, they must use a program such as Google Slides. This will allow the student to hand off the presentation to another person who can complete the task while they are away.
+
+*Tmux ≈ Google Slides + the team who will complete the task*
+
+
+## How To Create And Connect To Tmux
+
+1) Start a new session
+*Note: You can have as many tmux sessions running at a time! The work is saved and stays the same when you leave a particular session*
+
+![Create tmux session screenshot]({{ site.url }}/assets/tmux-01.jpg)
+
+![tmux pane screenshot]({{ site.url }}/assets/tmux-02.jpg)
+*The green line at the bottom of the screen tells the user they are in a tmux session (includes name of file and what is running, in this case bash)*
+
+2) You can split the screen horizontally and vertically
+
+__Vertical Split__ (Control-b and %)
+![tmux pane vertical split screenshot]({{ site.url }}/assets/tmux-03.jpg)
+
+__Horizontal Split__ (Control-b and ")
+![tmux horizontal split screenshot]({{ site.url }}/assets/tmux-04.jpg)
+
+3) Example: You can run something, detach, and it will continue running until it is complete
+
+__Run Bash Script__
+![tmux script screenshot]({{ site.url }}/assets/tmux-05.jpg)
+
+![tmux screenshot]({{ site.url }}/assets/tmux-06.jpg)
+*Note: Even if you detach, this will continue to run*
+
+__Detach__
+Use: Control-b and d; to exit the tmux session
+
+You will exit the session and see the message below
+![tmux exit session screenshot]({{ site.url }}/assets/tmux-07.jpg)
+
+You can check what sessions are running with `tmux ls`
+![tmux ls screenshot]({{ site.url }}/assets/tmux-08.jpg)
+
+__Reconnect__
+Use...
+![Back to tmux session screenshot]({{ site.url }}/assets/tmux-09.jpg)
+
+As you can see below the session kept running and finished the loop
+![tmux complete run screenshot]({{ site.url }}/assets/tmux-10.jpg)
+
+4) Use `exit` to leave the tmux session
+
+Once you exit, the session should no longer be listed
+![No tmux sessions listed screenshot]({{ site.url }}/assets/tmux-11.jpg)
+
+*Note: The files in tmux are the same as the ones outside of tmux. tmux is just a middle man!*
+![Middleman Internet Image](https://mk0industcomhmhlrip6.kinstacdn.com/wp-content/uploads/2015/02/middleman.jpg)
+
```

Comment:
The numbered list gets kind of lost behind the bolded Reconnect, Detach etc. 
Might want to reorganize the list or bold the numbers/indent the actions

Reply:
Got it, thanks!